src/modules/wifi/scan_hosts.cpp

```diff
@@ -5,140 +5,100 @@
 #include "core/wifi_common.h"
 #include "scan_hosts.h"
 #include "clients.h"
+#include <ESPping.h>
+#include <HTTPClient.h>
+#include <WiFi.h>
```

Comment:
ESPping.h and WiFi.h are already declared in scan_hosts.h

Reply:
Updated according to the principle include what you use